pydantic_ai_slim/pydantic_ai/agent/__init__.py

```diff
@@ -370,6 +370,14 @@ def __init__(
             _utils.Option[Sequence[Tool[AgentDepsT] | ToolFuncEither[AgentDepsT, ...]]]
         ] = ContextVar('_override_tools', default=None)
 
+        # Prompt overrides (experimental)
+        self._override_instructions: ContextVar[_utils.Option[str | None]] = ContextVar(
+            '_override_instructions', default=None
+        )
+        self._override_system_prompts: ContextVar[_utils.Option[tuple[str, ...]]] = ContextVar(
```

Comment:
Do we need this? Instructions seem to be the more modern version of system prompts, so unless you have a specific reason to want both, I'd rather only have instructions.